src/main/java/org/symphonyoss/symphony/messageml/elements/PersonSelector.java

```diff
@@ -14,28 +19,90 @@
 public class PersonSelector extends FormElement {
   public static final String MESSAGEML_TAG = "person-selector";
 
+  private static final String PLACEHOLDER_ATTR = "placeholder";
+
   private static final String PRESENTATIONML_TAG = "div";
+  private static final String PRESENTATIONML_PLACEHOLDER_ATTR = "data-placeholder";
+
   private final static String MARKDOWN = "Person Selector";
   private static final String CLASS_ATTR = "class";
+  private static final String PRESENTATIONML_NAME_ATTR = "data-name";
 
   public PersonSelector(Element parent) {
     super(parent, MESSAGEML_TAG);
   }
 
+  /*@Override
+  public void buildAll(MessageMLParser context, org.w3c.dom.Element element) throws InvalidInputException, ProcessingException {
+    switch (getFormat()) {
+      case MESSAGEML:
+        super.buildAll(context, element);
+        break;
+      case PRESENTATIONML:
+        buildElementFromDiv(context, element);
+        this.validate();
+        break;
+      default:
+        throw new InvalidInputException(String.format("Invalid message format for \"%s\" element", MESSAGEML_TAG));
+    }
+  }*/
+
+  private void buildElementFromDiv(MessageMLParser context, org.w3c.dom.Element element) throws InvalidInputException, ProcessingException {
+    if(!PRESENTATIONML_TAG.equals(element.getNodeName())) {
+      throw new InvalidInputException(String.format("Invalid PresentationML for the \"%s\" element", MESSAGEML_TAG));
+    }
+    buildNode(context, element.getAttributeNode(PRESENTATIONML_NAME_ATTR));
+    buildNode(context, element.getAttributeNode(PRESENTATIONML_PLACEHOLDER_ATTR));
+    
+  }
+
   @Override
   public void validate() throws InvalidInputException {
     super.validate();
-    assertNoAttributes();
+
+    if (getAttribute(NAME_ATTR) == null) {
+      throw new InvalidInputException("The attribute \"name\" is required");
+    }
+    
     assertNoContent();
   }
 
   @Override
   public void asPresentationML(XmlPrintStream out) {
-    out.printElement(PRESENTATIONML_TAG, null, CLASS_ATTR, MESSAGEML_TAG);
+    Map<String, String> presentationAttrs = buildPersonSelectorInputAttributes();
+    out.printElement(PRESENTATIONML_TAG, null, presentationAttrs);
   }
 
   @Override
   public Node asMarkdown() {
     return new FormElementNode(MARKDOWN);
```

Comment:
Consider putting the name of the field here (on the markdown) as well now that we have it